packages/perseus/src/types.ts

```diff
@@ -127,7 +127,7 @@ type TrackInteractionArgs = {
 } & Partial<TrackingGradedGroupExtraArguments> &
     Partial<TrackingSequenceExtraArguments>;
 
-export const MafsFlags = [
+export const MafsGraphTypeFlags = [
```

Comment:
This needed to split because we use this array in the dev Gallery to populate the graph type dropdown. I think it's nice to have it split out anyway.